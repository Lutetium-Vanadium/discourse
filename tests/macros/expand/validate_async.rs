fn main() {
    let q = inquisition::questions![Expand { async validate: todo!() }];
}
