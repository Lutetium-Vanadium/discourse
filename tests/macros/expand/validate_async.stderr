error: option `validate_async` does not exist for kind `expand`
 --> $DIR/validate_async.rs:2:52
  |
2 |     let q = inquisition::questions![Expand { async validate: todo!() }];
  |                                                    ^^^^^^^^
